### ISSUE TITLE
Only LiDAR, no camera

### ISSUE THREAD

**Issue by ZhangAlso** [HoEmpire/pedestrain_tracking_and_localizaiton#3]
Hello, can I use only LiDAR for detection?

**Comment by HoEmpire** [comment]
Yes. But only LiDAR won't be robust enough as it only uses clustering for detection, which could easily fail when multiple targets are close to each other.
